### PR TITLE
MEN-4746 Move to single db.

### PR DESCRIPTION
**Pull request by merlin-northern**
ChangeLog:title
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please

File: go.mod [hunk 1]
```diff
@@ -1,17 +1,64 @@
 module github.com/mendersoftware/deviceconfig
 
-go 1.15
+go 1.17
```
Comment (merlin-northern) [inline go.mod:3]: this I will revert to 1.15 as well as clean the indirects below.

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline sugar please

**Review by tranchitella — APPROVED** [review]
Works perfectly; I tried this PR in both OS and ENT, migrating from the previous version to the single-database one, and couldn't spot any issue.

**Review by tranchitella — APPROVED** [review]

**Comment by merlin-northern** [comment]
> Works perfectly; I tried this PR in both OS and ENT, migrating from the previous version to the single-database one, and couldn't spot any issue.

thank you!

**Pull request merged**